test/mix_test_watch/paths_test.exs

```diff
@@ -1,7 +1,7 @@
 defmodule MixTestWatch.PathTest do
   use ExUnit.Case
 
-  import MixTestWatch.Path, only: [watching?: 1, excluded?: 2]
+  import MixTestWatch.Path, only: [watching?: 1, watching?: 2, excluded?: 2]
```

Comment:
So - TIL: It looks like when I remove watching?: 1 leaving me with
```import MixTestWatch.Path, only: [watching?: 2, excluded?: 2] ```

I end up with a compiler error as though the default isn't picked up:
``` 
** (CompileError) test/mix_test_watch/paths_test.exs:7: undefined function watching?/1
    (stdlib) lists.erl:1337: :lists.foreach/2
    (stdlib) erl_eval.erl:669: :erl_eval.do_apply/6
```